internal/icinga2/api_responses.go

```diff
@@ -288,6 +289,14 @@ type ObjectCreatedDeleted struct {
 	EventType  string `json:"type"`
 }
 
+// IcingaAppStatus represents the Icinga 2 API status endpoint query result of type IcingaApplication.
+// https://icinga.com/docs/icinga-2/latest/doc/12-icinga2-api/#status-and-statistics
+type IcingaAppStatus struct {
+	App struct {
+		EnableFlapping bool `json:"enable_flapping"`
+	} `json:"app"`
+}
```

Comment:
This does not match the structure returned by that API endpoint:

```json
{
  "results": [
    {
      "name": "IcingaApplication",
      "perfdata": [],
      "status": {
        "icingaapplication": {
          "app": {
            "enable_event_handlers": true,
            "enable_flapping": true,
            "enable_host_checks": true,
            "enable_notifications": true,
            "enable_perfdata": true,
            "enable_service_checks": true,
            "environment": "",
            "node_name": "master-1",
            "pid": 19570,
            "program_start": 1721306330.03043,
            "version": "v2.14.0-206-gb468e0cee"
          }
        }
      }
    }
  ]
}
```

Thus, `EnableFlapping` always remains `false`.